src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,24 @@
-export const TodoInfo = () => {};
+import React from 'react';
+
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    title,
+    completed,
+    user,
+  } = todo;
+
+  return (
+    <article className={`TodoInfo ${completed && 'TodoInfo--completed'}`} data-id={todo.id}>
```

Comment:
use `classnames` lib for it
```suggestion
    <article 
      className={cn(....)} 
      data-id={todo.id}
    >
```